api-tests/specs/createTrialSite/createNewTrialSite.js

```diff
@@ -36,12 +36,12 @@ describe('As a user with Create Trial Sites permission I want to have the system
         countryParentSiteId = parseCountryParentSiteResponseData.id
     });
 
-    it('GIVEN I have Trial Site Type fields and site structure rules defined AND I have an existing trial site to act as the ‘parent’ (e.g. top node in the tree) WHEN I submit an API request to create a Trial site with a value for all mandatory fields, a unique name and a permitted parent trial site ‘type’ with no fields exceeding their specified maximum length THEN AC1 a new Trial site record is created in the system (added to the Site structure as a child of its parent) with its parent identifier, a unique identifier for itself and an acknowledgement is returned', async () => {
-        let countryAsParent = requests.validSiteLCC
-        countryAsParent.parentSiteId = countryParentSiteId
-        const lccResponseId = await baseRequest.post(endpointUri).send(countryAsParent);
-        expect(lccResponseId.status).to.equal(HttpStatus.CREATED)
-    });
+    // it('GIVEN I have Trial Site Type fields and site structure rules defined AND I have an existing trial site to act as the ‘parent’ (e.g. top node in the tree) WHEN I submit an API request to create a Trial site with a value for all mandatory fields, a unique name and a permitted parent trial site ‘type’ with no fields exceeding their specified maximum length THEN AC1 a new Trial site record is created in the system (added to the Site structure as a child of its parent) with its parent identifier, a unique identifier for itself and an acknowledgement is returned', async () => {
```

Comment:
Instead of commenting this, why not add address to the LCC thats being created, wdyt ?

Reply:
It was failing with compilation errors when I added an address entity to the LCC, have checked in the changes now. Need to check if it works.

Reply:
Its working Saj :)